src/core/expression/qgsexpressionnodeimpl.cpp

```diff
@@ -99,6 +99,8 @@ QString QgsExpressionNode::NodeList::cleanNamedNodeName( const QString &name )
     cleaned = QStringLiteral( "geometry1" );
   else if ( cleaned == QLatin1String( "geometry b" ) )
     cleaned = QStringLiteral( "geometry2" );
+  else if ( cleaned == QLatin1String( "i" ) )
```

Comment:
Are we sure this is OK for all parameter called `i` ? There are all vertex ?

Reply:
Yes @DelazJ checked here https://github.com/qgis/QGIS/pull/50853#issuecomment-1310395168, we shouldn't add an `i` parameter anymore.